Documentation/colombia_schema.md

```diff
@@ -0,0 +1,30 @@
+
+## schema = colombia
+
+In the schema `colombia` we have datasets related to this country. Up to now, the datasets available are related to the COVID-19 and stratified by departamento<sup>1</sup>. They were provided by the **Instituto Nacional de Salud**. In the table below, we have a brief description of the content of each dataset. For each of them, we have another dataset named `{dataset_name}_meta` that contains an explanation of the columns in the dataset.
+
+For example, in the schema `colombia` we have a dataset called `casos_positivos_covid` that represents the daily records timelines for positive cases in each `departamento`<sup>1</sup> of Colombia. The meaning of the columns of this dataset are explained in the `casos_positivos_covid_meta` dataset.
+
+**departamento<sup>1</sup>** - refers to a political division of Colombia.
+
+**ATTENTION:**
```

Comment:
I thought I might have an explanation of what that word means, for people who don't know much about the country and its division. But if you think it is not necessary I can remove it.

Reply:
In the `casos_positivos_covid_meta` dataset there is an explanation of each of the columns of the table. But I would like to highlight the meaning of these 3 columns for the non-spanish speakers because they are shown in the column `key` of the table below. 

In the metadata table, I put an explanation, in English, of the Spanish terms that appear in the table.

Reply:
I think the English explanation in the metadata table should be enough.

Reply:
Ok